zproject/test_extra_settings.py

```diff
@@ -17,6 +17,9 @@
     LOGGING,
 )
 
+if TYPE_CHECKING:
+    from zerver.lib.types import OIDCIdPConfigDict
+
```

Comment:
A few lines above this we have `from zerver.lib.types import SAMLIdPConfigDict, SCIMConfigDict` -  should these be also moved into this `if TYPE_CHECKING` block here analogically?

Reply:
Actually, I don' remember why that `if TYPE_CHECKING` thing is necessary here.